consensus/ibft/src/main/java/tech/pegasys/pantheon/consensus/ibft/statemachine/RoundChangeManager.java

```diff
@@ -137,14 +137,14 @@ private RoundChangeStatus storeRoundChangeMessage(final SignedData<RoundChangePa
    *
    * @param completedRoundIdentifier round identifier that has been identified as superseded
    */
-  public void discardCompletedRound(final ConsensusRoundIdentifier completedRoundIdentifier) {
+  public void discardRoundsPriorTo(final ConsensusRoundIdentifier completedRoundIdentifier) {
     roundChangeCache
         .entrySet()
-        .removeIf(entry -> isAnEarlierOrEqualRound(entry.getKey(), completedRoundIdentifier));
+        .removeIf(entry -> isAnEarlierRound(entry.getKey(), completedRoundIdentifier));
```

Comment:
not part of your change, but can simplify with `.keySet().removeIf(roundIdentifier-> isAnEarlierRound(roundIdentifier, ...`

Reply:
Done.